### PR TITLE
getYouTubeVideoID ~ TV support

### PR DESCRIPTION
**Pull request by OfficialNoob**

### CODE REVIEW

**Comment by ajayyy** [comment]
Nice! That was fast.

File: utils.js [hunk 1]
```diff
@@ -32,7 +35,8 @@ function getYouTubeVideoID(url) {
     if(!["www.youtube.com","www.youtube-nocookie.com"].includes(urlObject.host)) return false; 
 
     //Get ID from searchParam
-    if ((urlObject.pathname == "/watch" || urlObject.pathname == "/watch/") && urlObject.searchParams.has("v")) {
+    if(!urlObject.searchParams.has("v")) return false
```
Comment (ajayyy) [inline utils.js]: This breaks embeds right?

**Comment by ajayyy** [comment]
Looks good 👍 . Ready to merge?

**Comment by OfficialNoob** [comment]
Yes

**Pull request merged**